lang/python/python-zmq/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (C) 2017-2021 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-zmq
+PKG_VERSION:=22.0.3
+PKG_RELEASE:=1
+
+PYPI_NAME:=pyzmq
+PKG_HASH:=f7f63ce127980d40f3e6a5fdb87abf17ce1a7c2bd8bf2c7560e1bbce8ab1f92d
+
+PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec@nic.cz>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=COPYING.BSD
+
+PKG_BUILD_DEPENDS:=cython
```

Comment:
this `PKG_BUILD_DEPENDS` doesn't do anything AFAICT;
we have a `HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:=Cython` for some packages like `numpy`

Reply:
According to the [project wiki](https://github.com/zeromq/pyzmq/wiki/Building-and-Installing-PyZMQ#building-pyzmq-from-source), Cython is not required to build pyzmq from a release package.